build/build.sh

```diff
@@ -35,12 +35,12 @@ pandoc --verbose \
   --defaults=common.yaml \
   --defaults=html.yaml
 
-# Return null if docker command is missing, otherwise return path to docker
-DOCKER_EXISTS="$(command -v docker || true)"
+# Set DOCKER_RUNNING to a non-empty string if docker is running, otherwise null.
+DOCKER_RUNNING="$(docker info &> /dev/null && echo "yes" || true)"
```

Comment:
Parts of this command:

1. `docker info &> /dev/null`: run docker info for exit status only, discarding both stdout and stderr
2. `&& echo "yes"`: if part 1 was a success, print "yes" to std, to set `DOCKER_RUNNING=yes`
3. `|| true` if part 1 returned a failing exit status still return a zero (success) status